Assets/Scripts/UI/Login/UILoginMenu.cs

```diff
@@ -319,29 +319,37 @@ private static void PreventDeselection(ref GameObject previouslySelectedGameObje
         /// <returns>True if input should be handled, false if not.</returns>
         private static bool ShouldInputBeHandled()
         {
-            // TODO: Clarify why this is only evaluated for PS4 and PS5
-#if UNITY_PS4 || UNITY_PS5
-            // TODO: Simplify this conditional - it's sort of confusing to read despite doing exactly what we want
-            if (null == EventSystem.current || EventSystem.current.sendNavigationEvents == !EOSManager.Instance.IsOverlayOpenWithExclusiveInput())
```

Comment:
I think we still need to check if `EventSystem.current` is `null`. While it was far from perfect, this conditional _did_ provide safety against referencing a null pointer.

Reply:
Agreeable; it seems like we're using EventSystem.current without checking that it exists, later in this function.

Reply:
Fixed